bouncer/shared/create_and_delete_multiple_orders.ts

```diff
@@ -1,54 +1,97 @@
+import assert from 'assert';
+import { ApiPromise } from '@polkadot/api';
 import { Keyring } from '../polkadot/keyring';
 import { handleSubstrateError, lpMutex } from '../shared/utils';
 import { getChainflipApi, observeEvent } from './utils/substrate';
 import { limitOrder } from './limit_order';
+import { rangeOrder } from './range_order';
+import { depositLiquidity } from './deposit_liquidity';
+import { deposits } from './setup_swaps';
 
-export async function createAndDeleteMultipleOrders(numberOfOrders: number) {
+const LP: string = '//LP_3';
+
+async function countOpenOrders(
+  baseAsset: string,
+  quoteAsset: string,
+  lp: string,
+  chainflip: ApiPromise,
```

Comment:
There's no need to pass in the API here, can just initialise it with `await using api = await getChainflipApi();`  - the initialiser of that function ensures there's only one underlying client and we effectively just get a reference to it.